opennlp-tools/src/main/java/opennlp/tools/util/wordvector/Glove.java

```diff
@@ -29,6 +29,9 @@
 import opennlp.tools.util.java.Experimental;
 
 /**
+ *
+ * GloVe is an unsupervised learning algorithm for obtaining vector representations for words.
+ * Details are found on this <a href="https://nlp.stanford.edu/projects/glove/"> web site</a>.
```

Comment:
extra space inside the `<a>` tag but I think it will be correctly formatted by the browser.

Reply:
Will fix, once the "CC0" topic is discussed and decided.